examples/mcp-for-ai-coding-assistants/README.md

```
@@ -0,0 +1,606 @@
+Configure MCP servers (the [Atmos MCP server](/ai/mcp-server) plus the
+[AWS MCP server suite](https://github.com/awslabs/mcp)) **once** in `atmos.yaml`, then use
+the same set of tools — with the same AWS credentials managed by Atmos Auth — from
+[Claude Code](https://www.anthropic.com/claude-code),
+[OpenAI Codex CLI](https://github.com/openai/codex), and
+[Google Gemini CLI](https://github.com/google-gemini/gemini-cli).
+
+Learn more in the [MCP configuration documentation](/cli/configuration/mcp).
+
+---
+
+### Your AI knows your stacks and components. And your cloud. And your history.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)